src/server.jsx

```diff
@@ -115,6 +114,10 @@ server
           res.set({
             'Cache-Control': 'public, max-age=60, no-transform',
           });
+
+          // Displays error in console
+          console.error(error);
```

Comment:
@sneridagh @robgietema we might want to think about a generic way about how to handle errors in Volto. Maybe factor that out into a function so that we can enhance it easily at a later point. Has nothing to do with this PR directly. Just a thought that crossed my mind.